src/main/java/de/tum/in/www1/artemis/service/ModelAssessmentKnowledgeService.java

```diff
@@ -12,6 +14,8 @@
 @Service
 public class ModelAssessmentKnowledgeService {
 
+    private final Logger log = LoggerFactory.getLogger(ExerciseService.class);
```

Comment:
I think you can/should remove the Logger now :)

Reply:
I have removed it on my last commit :)